ambari/administration/admin-tasks.rst

```diff
@@ -0,0 +1,109 @@
+Ambari Admin Tasks
+==================
+
+An "Ambari Admin" has administrator (or super-user) privilege. When logged into Ambari with the "Ambari Admin" privilege, you can:
+
+* :ref:`Change the Administrator Account Password<ambari-change-admin-password>`
+* :ref:`Create a cluster<ambari-creating-cluster>`
+* :ref:`Set access permissions for an existing cluster<ambari-cluster-permission>`
```

Comment:
Section title is _Setting Cluster Permissions_ while in index is _Set access permissins for an existing cluster_. Use same text.